src/Lime.Protocol/Network/EnvelopePipe.cs

```diff
@@ -0,0 +1,237 @@
+#if NETSTANDARD2_1
+using System;
+using System.Buffers;
+using System.IO.Pipelines;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Lime.Protocol.Serialization;
+using Lime.Protocol.Server;
+
+namespace Lime.Protocol.Network
+{
+    public sealed class EnvelopePipe : IStartable, IStoppable, IDisposable
```

Comment:
Maybe it would be nice to include some xml summaries in this class, as it doesn't have any specific documented interface.